src/ocean/util/app/ext/VersionArgsExt.d

```diff
@@ -230,7 +209,10 @@ class VersionArgsExt : IApplicationExtension, IArgumentsExtExtension,
 
     public override void setupArgs ( IApplication app, Arguments args )
     {
-        args("version").params(0).help("show version information and exit");
+        args("version").params(0)
+            .help("show version information and exit");
+        args("build-info").params(0)
+            .help("show detailed build information and exit");
```

Comment:
Should these arguments conflict? If not maybe it should be mentioned that if both are found, then `--build-info` will be used (if I understand correctly, give the `if` order, that's what will happen).

Reply:
`--build-info` is an exact superset of `--version` (prints the same stuff + more of its own), so it makes sense to make you can supply both

Reply:
For me it would make more sense to be able to use both it wasn't a super-set (so using both would print more info than only using one). For me exactly because it is a superset is that using both would look super weird. But it's just a minor thing...